compiler/ballerina-lang/src/test/java/org/ballerinalang/toml/ManifestProcessorTest.java

```diff
@@ -149,21 +184,53 @@ public void testSingleEmptyDependencies() throws TomlException {
         Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
     }
 
-    @Test(description = "Multiple dependencies added to the dependencies section has an effect")
-    public void testMultipleDependencies() throws TomlException, IOException {
-        Path tmpDir = Files.createTempDirectory("manifest-test-");
-        Path baloPath = tmpDir.resolve("string_utils.balo");
-        Files.createFile(baloPath);
-        
-        Manifest manifest = ManifestProcessor.parseTomlContentFromString(this.validProjectBlock + "[dependencies] \n " +
-                "string-utils = { path = \"" + baloPath + "\", version = \"1.0.5\" } \n " +
-                "jquery = { version = \"2.2.3\" } \n");
-        Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
-        Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getVersion(), "1.0.5");
-        Assert.assertEquals(manifest.getDependencies().get(1).getModuleID(), "jquery");
-        Assert.assertEquals(manifest.getDependencies().get(1).getMetadata().getVersion(), "2.2.3");
-    
-        Files.delete(baloPath);
-        Files.delete(tmpDir);
+    @Test(description = "Multiple dependencies added to the dependencies section has an effect except on Windows")
+    public void testMultipleDependenciesLinux() throws TomlException, IOException {
+        if (OS.contains("win")) {
+            flag = false;
+        }
+        if (flag) {
+            Path tmpDir = Files.createTempDirectory("manifest-test-");
+            Path baloPath = tmpDir.resolve("string_utils.balo");
+            Files.createFile(baloPath);
+
+            String tomlData = this.validProjectBlock + "[dependencies] \n string-utils = { path = \""
+                    + baloPath + "\", version = \"1.0.5\" } \n "
+                    + "jquery = { version = \"2.2.3\" } \n";
+
+            Manifest manifest = ManifestProcessor.parseTomlContentFromString(tomlData);
+            Assert.assertEquals(manifest.getDependencies().get(0).getModuleID(), "string-utils");
+            Assert.assertEquals(manifest.getDependencies().get(0).getMetadata().getVersion(), "1.0.5");
+            Assert.assertEquals(manifest.getDependencies().get(1).getModuleID(), "jquery");
+            Assert.assertEquals(manifest.getDependencies().get(1).getMetadata().getVersion(), "2.2.3");
+
+            Files.delete(baloPath);
+            Files.delete(tmpDir);
+        }
+    }
+
+    @Test(description = "Multiple dependencies added to the dependencies section has an effect only on Windows")
+    public void testMultipleDependenciesWindows() throws TomlException, IOException {
+        if (!OS.contains("win")) {
```

Comment:
We can have same test for both windows and linux since we use resolve to get the path of the balo and skip the test if it is windows.